pkg/controller/appdefinition/volume.go

```diff
@@ -206,6 +209,55 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 	return
 }
 
+func GetPVForVolumeBinding(req router.Request, appInstance *v1.AppInstance, binding v1.VolumeBinding) (*corev1.PersistentVolume, error) {
```

Comment:
nit: this looks very specific to this implementation. Should this be unexported?

Reply:
This confused me during review. Actually @tylerslaton asked for it to be exported as he could use it elsewhere: https://github.com/acorn-io/acorn/pull/1615#discussion_r1199160483